ios/FluentUI/Pill Button Bar/PillButtonBar.swift

```diff
@@ -137,8 +139,9 @@ open class PillButtonBar: UIScrollView {
         }
     }
 
-    @objc public init(pillButtonStyle: PillButtonStyle = .outline) {
+    @objc public init(pillButtonStyle: PillButtonStyle = .outline, pillButtonBackgroundColor: UIColor? = nil) {
```

Comment:
this might be ugly in @objc interface. I suggest creating another init() so that existing pillbar client don't need to update their caller

Reply:
Makes sense! @DeepaliNemade839 could you create an extra initializer?

Reply:
added an additional initializer

---
In reply to: [468772123](https://github.com/microsoft/fluentui-apple/pull/175#discussion_r468772123) [](ancestors = 468772123)

Reply:
How about a convenience initializer to avoid duplication of their logic?

    @objc public convenience init(pillButtonStyle: PillButtonStyle = .outline) {
		self.init(pillButtonStyle: pillButtonStyle, pillButtonBackgroundColor: nil)
    }

Reply:
Added a convenience initializer

---
In reply to: [468767678](https://github.com/microsoft/fluentui-apple/pull/175#discussion_r468767678) [](ancestors = 468767678)

Reply:
Done. Learning swift. This was new for me :)

---
In reply to: [468846303](https://github.com/microsoft/fluentui-apple/pull/175#discussion_r468846303) [](ancestors = 468846303)

Reply:
Thank you for making this change! :)